core/src/trezor/ui/layouts/tt/webauthn.py

```diff
@@ -13,17 +13,24 @@ async def confirm_webauthn(
     ctx: wire.GenericContext | None,
     info: ConfirmInfo,
     pageable: Pageable | None = None,
-) -> bool:
+) -> tuple[bool, int]:
```

Comment:
Wouldn't `int | None` be a more pythonic choice? Or just int and exception when not confirmed.

Reply:
Changed to this in [f44f512](https://github.com/trezor/trezor-firmware/pull/2556/commits/f44f5126ddf39a27ba035f698f2512abf0ea72c8)